singer/pom.xml

```diff
@@ -190,11 +190,17 @@
             <version>1.12.248</version>
         </dependency>
         <dependency>
-		    <groupId>com.salesforce.kafka.test</groupId>
-		    <artifactId>kafka-junit4</artifactId>
-		    <version>3.2.0</version>
-		    <scope>test</scope>
-		</dependency>
+            <groupId>com.salesforce.kafka.test</groupId>
```

Comment:
This dependency is under BSD license whereas Singer is Apache license

Reply:
is there a workaround for this? we use this for the unit test simulating broker/broker down.

Reply:
How come the dependency already exists in the repo? Did we miss this earlier?